### DISCUSSION TITLE
NFT Challenges

### DISCUSSION THREAD

**Discussion by cbdevsec** [Cyfrin/blockchain-basics-cu#3] [General]
hello, I have finished the course but I don't know what am I supposed to do with the last 2 challenges. am I supposed to debug something? or verify somethig?

**Comment by suchapalaver** [comment]
Same.

**Comment by KMean** [comment]
Go trough the code on etherscan and you should be able to understand what you have to do 😉

**Comment by cmariscalvo** [comment]
i am trying to call the method solveChallenge on Sepolia for the NFT contract but it is not working: 
![image](https://github.com/user-attachments/assets/8390cc45-cd9b-4657-99ab-fb804e1f8021)
